hack/release.sh

```diff
@@ -120,6 +120,20 @@ function build_images() {
     --build-arg "GIT_COMMIT=${VERSION}" \
     .
 
+     #-t "sabu-persistence-service-docker-local.artifactory.eng.vmware.com/nigupta/vsphere-csi-win":"win-1"  
+   echo "building ${CSI_IMAGE_NAME}-windows:${VERSION}"
+   echo "GOPROXY=${GOPROXY}"
+   #CSI_IMAGE_NAME_WIN="sabu-persistence-service-docker-local.artifactory.eng.vmware.com/nigupta/vsphere-csi"
```

Comment:
Can you remove the internal link and commented lines?

Reply:
done